src/main/java/svydovets/core/context/DefaultApplicationContext.java

```diff
@@ -99,4 +102,26 @@ public <T> Map<String, T> getBeansOfType(Class<T> requiredType) {
                 .filter(entry -> requiredType.isAssignableFrom(entry.getValue().getClass()))
                 .collect(Collectors.toMap(Map.Entry::getKey, entry -> requiredType.cast(entry.getValue())));
     }
+
+    private void populateProperties(){
+        for (Map.Entry<String, Object> entry : beanMap.entrySet()) {
+            Object object = entry.getValue();
+            Field[] fields = object.getClass().getDeclaredFields();
+
+            for (Field field : fields) {
+                boolean annotationPresent = field.isAnnotationPresent(Autowired.class);
+
+                if (annotationPresent) {
+                    Object autowireCandidate = getBean(field.getType());
+                    try {
+                        field.setAccessible(true);
+                        field.set(object, autowireCandidate);
+                    } catch (IllegalAccessException e) {
+                        e.printStackTrace();
+                    }

```

Comment:
1) Please, create an additional method and move try catch block there, for example:
"setDependency(field, autowireCandidate)"
2) Please, throw a custom exception, for example, "AutowireBeanException" with message like in "createBean()" method

Reply:
changes were added